src/analysis/syntax_doc.ml

```diff
@@ -219,7 +219,7 @@ let get_syntax_doc node : syntax_info =
              sub-expression and replaced by a fresh type variable.";
           documentation = syntax_doc_url "locallyabstract.html";
         }
-  | (_, Module_expr _) :: (_, Module_binding _) :: _ ->
+  | (_, Module_expr _) :: (_, Module_expr _) :: _ ->
```

Comment:
This is unrelated to the commit message, it should not be here. (No need to change it, we will probably squash most of the commits at the end anyway.)